### PR TITLE
Update LTS to 14.4

### PR DESCRIPTION
**Pull request by vladciobanu**
###### Reviewer checklist

- [ ] Test coverage: `stack test --coverage`
- [ ] Public API documentation: `stack haddock`
- [ ] Style conformance: `stylish-haskell`

---

### CODE REVIEW

**Comment by vladciobanu** [comment]
@ttuegel Any idea wny the build fails with `stylish-haskell: No such file or directory`?

**Comment by vladciobanu** [comment]
It looks like it's fixed now, but I have no idea how this ever worked.

File: scripts/stylish.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ export TOP=${TOP:-$(git rev-parse --show-toplevel)}
 
 source $TOP/scripts/run-on-haskell.include.sh
 
-stack build stylish-haskell
+stack install stylish-haskell
 export PATH=$(stack path --bin-path)
```
Comment (ttuegel) [inline scripts/stylish.sh]: Setting `PATH` should no longer be necessary, if we are installing `stylish-haskell` into the `local-bin-path`.

File: stack.yaml [hunk 2]
```diff
@@ -47,7 +47,6 @@ packages:
 # (e.g., acme-missiles-0.3)
 extra-deps:
   - graphviz-2999.20.0.3@sha256:cde383c356bc41136ed53cd27e0800f46dbd2185600dd0de18d66d5c49739d94
-  - tasty-hedgehog-0.2.0.0@sha256:83a8b777fa472040979e44dba43c32441f55d5ddb9641a4d53deee4b0e09fa34
```
Comment (ttuegel) [inline stack.yaml]: :tada:

**Pull request merged**